db/models.py

```diff
@@ -14,3 +15,36 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField(null=True)
+    actors = models.ManyToManyField(Actor, related_name="movies")
+    genres = models.ManyToManyField(Genre, related_name="movies")
+
+    def __str__(self) -> str:
+        return self.title
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField(default=0)
+    seats_in_row = models.IntegerField(default=0)
+
+    def __str__(self) -> str:
+        return self.name
+
+    @property
+    def capacity(self) -> int:
+        return self.rows * self.seats_in_row
+
+
+class MovieSession(models.Model):
+    show_time = models.DateTimeField(default=datetime.now())
```

Comment:
Using default value with the call here lays several limitations on the DB and may not have an expected behavior. Use `auto_now_add ` here for this purpose.

Reply:
when I try to use auto_now_add=  half of the tests does not pass

Reply:
![Screenshot from 2023-09-27 21-33-54](https://github.com/mate-academy/py-movie-moviesession-cinemahall/assets/122517203/f198421f-e520-4bbd-ae9d-29807a00eb9f)
Related to this task you don't need add any parameters to DateTimeField because you should add datetime to movie session manually